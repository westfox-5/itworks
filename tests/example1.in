4 4
3
1 2 233
2 3 150
0 2 70
2
0 1 23
1 0 30