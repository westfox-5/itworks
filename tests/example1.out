3
1 2 233
2 3 150
0 2 70

